static-site/package.json

```diff
@@ -31,6 +31,7 @@
     "heroku-postbuild": "gatsby build"
   },
   "dependencies": {
+    "@reach/router": "^1.2.1",
```

Comment:
This dep should be dropped -- it was only used in `withLocation` which was removed in 26cbaa2f132cd164ea0e3635f12683726139819f.

Note also that gatsby v1 uses `react-router` and so adding this dep introduces the possibility of unnecessary library duplication. Moving to gatsby v2+ will give access to @reach/router out of the box.